Library/ViewModels/ManagePledgePaymentMethodViewModelTests.swift

```diff
@@ -0,0 +1,41 @@
+@testable import KsApi
+@testable import Library
+import Prelude
+import ReactiveExtensions_TestHelpers
+import ReactiveSwift
+import XCTest
+
+final class ManagePledgePaymentMethodViewModelTests: TestCase {
+  internal let vm: ManagePledgePaymentMethodViewModelType = ManagePledgePaymentMethodViewModel()
+
+  private let cardImage = TestObserver<String, Never>()
+  private let cardNumberAccessibilityLabel = TestObserver<String, Never>()
+  private let cardNumberTextShortStyle = TestObserver<String, Never>()
+  private let expirationDateText = TestObserver<String, Never>()
+
+  internal override func setUp() {
+    super.setUp()
+
+    self.vm.outputs.cardImage.observe(self.cardImage.observer)
+    self.vm.outputs.cardNumberAccessibilityLabel.observe(self.cardNumberAccessibilityLabel.observer)
+    self.vm.outputs.cardNumberTextShortStyle.observe(self.cardNumberTextShortStyle.observer)
+    self.vm.outputs.expirationDateText.observe(self.expirationDateText.observer)
+  }
+
+  func testPaymentSourceInfo() {
+    self.vm.inputs.configureWith(value: Backing.PaymentSource.template)
+
+    self.cardImage.assertLastValue("icon--visa")
+    self.cardNumberAccessibilityLabel.assertLastValue("Visa, Card ending in 1111")
+    self.cardNumberTextShortStyle.assertLastValue("Ending in 1111")
+    self.expirationDateText.assertLastValue("Expires 09/2019")
+  }
+
+  func testApplePay() {
+    self.vm.inputs.configureWith(value: Backing.PaymentSource.applePay)
+
+    self.cardImage.assertValue("icon--apple-pay")
+    self.cardNumberTextShortStyle.assertLastValue("Ending in 1111")
```

Comment:
Are we missing an assertion here for the accessibility label? And actually on second thought, do we want to update the accessibility label to indicate that the card is associated with Apple Pay?

Reply:
Regarding the accessibility label, it's a good idea, but I think this ends up being out of scope for this PR.